src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/WorkflowService.scala

```diff
@@ -191,6 +196,26 @@ class WorkflowServiceImpl(override val workflowRepository: WorkflowRepository,
     } yield WorkflowImportExportWrapper(workflow, jobTemplates)
   }
 
+  override def exportWorkflowsAsZip(workflowIds: Seq[Long])(implicit ec: ExecutionContext): Future[ByteArrayOutputStream] = {
+    val workflowExportsFut = Future.sequence(workflowIds.map(exportWorkflow(_)))
```

Comment:
I would rather do it with one call to db. (or two, one for workflows and one for templates)

Reply:
ok